support/helpers/spreadsheet_data.rb

```diff
@@ -13,6 +13,10 @@ def create(valid_item_count, invalid_item_count)
     invalid_item_count.times { @data << invalid_mediator.values }
   end
 
+  def practice_data
```

Comment:
Yeap, it's raw data, injected into the table of data specified in the Cucumber. It's because telephone number is the only part of the practice data that has been validated so far. The remaining 3 parts will come in future user stories. 

We may want a cuke specifically for practice data. Once all parts of a practice have validations, it may become clearer on the need for cuke or not.